contracts/commands/AaveStopLossCommandV2.sol

```diff
@@ -17,7 +17,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.8.13;
+pragma solidity ^0.8.0;
```

Comment:
shouldn't we fix the version ?

Reply:
all other commands have
pragma solidity ^0.8.0;

so I wanted to keep it consistent

also 
^0.8.0;

means 0.8.X

so once we let say upgrade 0.8.14 wi will not need to change AAveStopLoss